minecraft_servers/zelicraft/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "server_name": "zelicraft",
+  "nice_name": "Zelicraft",
+  "direct_ip": "zelicraft.com",
+  "server_wildcards": [
+    "dev.zelicraft.com",
+    "potyh.es",
+    "followersland.me",
+    "endydev.com"
+  ],
+  "supported_languages": [
+     "es",
+     "en"
+  ],
+  "social": {
+    "web": "https://zelicraft.com",
```

Comment:
1. We do not support default nginx pages as web page ;)
2. https not supported

Reply:
I already changed it.